rust/arrow/src/buffer.rs

```diff
@@ -21,6 +21,7 @@
 use packed_simd::u8x64;
 
 use std::cmp;
+use std::convert::AsRef;
```

Comment:
nit: this is not used?

Reply:
This seems to be used by other parts. Anyway, cargo check doesn't complain about this.

Reply:
I see. I think it's in the preclude so no need to explicitly import previously.